tests/js/session-tests.js

```diff
@@ -803,10 +803,12 @@ module.exports = {
                         schema: [{ name: 'Dog', properties: { name: 'string' } }]
                     };
 
+                    Realm.deleteFile(config);
```

Comment:
why do we need to do this. I don't think we have the file there but it does not hurt to leave it just to be sure